docs/document/content/dev-manual/encrypt.en.md

```diff
@@ -5,20 +5,22 @@ weight = 12
 chapter = true
 +++
 
-## SPI Interface
+## EncryptAlgorithm
 
-| SPI Name         | Description            |
-| ---------------- | ---------------------- |
-| EncryptAlgorithm | Data encrypt algorithm |
+### Fully-qualified class name
 
-## Sample
+[`org.apache.shardingsphere.encrypt.algorithm`](https://github.com/apache/shardingsphere/blob/master/shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-api/src/main/java/org/apache/shardingsphere/encrypt/spi/EncryptAlgorithm.java)
 
-### EncryptAlgorithm
+### Definition
 
-| *Implementation Class*        | *Description*              |
-| ----------------------------- | -------------------------- |
-| MD5EncryptAlgorithm           | MD5 data encrypt algorithm |
-| AESEncryptAlgorithm           | AES data encrypt algorithm |
-| RC4EncryptAlgorithm           | RC4 data encrypt algorithm |
-| SM3EncryptAlgorithm           | SM3 data encrypt algorithm |
-| SM4EncryptAlgorithm           | SM4 data encrypt algorithm |
+Data encryption algorithm definition
+
+### Implementation classes
+
+| *Configuration Type* | *Description*                             | *Fully-qualified class name* |
+| -------------------- | ----------------------------------------- | ---------------------------- |
+| MD5EncryptAlgorithm           | MD5 data encrypt algorithm | [`org.apache.shardingsphere.encryption.algorithm.MD5Encrypt`](https://github.com/apache/shardingsphere/blob/master/shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/algorithm/MD5EncryptAlgorithm.java) |
```

Comment:
MD5EncryptAlgorithm should be MD5, please reference the `getType` of source code. Other rows are same

Reply:
done

Reply:
Great